lib/redux/actions/cash_wallet_actions.dart

```diff
@@ -147,6 +152,9 @@ ThunkAction startBalanceFetchingCall() {
         t.cancel();
         return;
       }
+      tokenAddress = store.state.cashWalletState.token == null
+        ? null
+        : store.state.cashWalletState.token.address;
```

Comment:
cant you do:
tokenAddress = store.state.cashWalletState.token?.address
instead?

Reply:
yeah looks like that exactly what I need